features/support/ui/landing.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'ui'
+
+module UI
+  class Landing < SitePrism::Page
+    set_url '/'
+    set_url_matcher(/wpcc\/\d+\/landing/)
+
+    element :age, '#your_details_form_age'
+    element :gender, '#your_details_form_gender'
+    element :salary, '#your_details_form_salary'
+    element :minimum_contribution, 'your_details_form_calculate_minimum'
```

Comment:
This is a radio option so the `element :full_contribution` should be added also.